File: 백준/Silver/1316. 그룹 단어 체커/그룹 단어 체커.py
def is_con(word):
   for i in range(len(word)-1):
      if word[i]==word[i+1]:
         continue
      elif word[i] in word[i+1:]:
         return int(False)
   return int(True)



result=0
for test_case in range(int(input())):
   result+=is_con(input())
print(result)

File: 백준/Silver/1316. 그룹 단어 체커/그룹 단어 체커.c
#include <stdio.h>

int check(char *word)
{
    for(int i=0;word[i];i++)
    {
        for(int j=i+1;word[j];j++)
        {
            if(word[j]!=word[j-1] && word[i]==word[j]) return 0; //앞글자랑 다른데 이미있는글자
        }
    }
    return 1;
}

int main(void)
{
    int num_word;
    int result=0;
    scanf("%d",&num_word);
    char word[110];
    for(int i=0;i<num_word;i++)
    {
        scanf("%s",word);
        result+=check(word);
    }
    printf("%d",result);
    return 0;
}